contracts/Strategy.sol

```diff
@@ -41,19 +51,23 @@ contract Strategy is BaseStrategy, IERC721Receiver {
     IDInterest public pool;
     // nft for redeeming mph that vests linearly
     IVesting public vestNft;
-    bytes constant internal DEPOSIT = "deposit";
-    bytes constant internal VEST = "vest";
+    bytes internal constant DEPOSIT = "deposit";
+    bytes internal constant VEST = "vest";
     uint64 public depositId;
     uint64 public maturationPeriod;
     address public oldStrategy;
 
     // Decimal precision for withdraws
-    uint public minWithdraw;
+    uint256 public minWithdraw;
     bool public allowEarlyWithdrawFee;
 
-    uint constant internal basisMax = 10000;
+    uint256 internal constant basisMax = 10000;
     IERC20 public reward;
-    uint constant private max = type(uint).max;
+    uint256 private constant max = type(uint256).max;
+    uint256 internal constant oneHundredPercentInBips = 10000;
```

Comment:
small change, basisMax is meant to be 100% in bips so can be reused

Reply:
Fixed